src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php

```diff
@@ -161,16 +161,16 @@ private function connect()
 
         $this->connection = ldap_connect($this->config['connection_string']);
 
+        if (false === $this->connection) {
```

Comment:
We should assign to the property only if a valid result is returned.
Also `ldap_error($connection)` cannot work on `false`.
Looking at [the doc](https://www.php.net/manual/en/function.ldap-connect.php), the error message is wrong also...

Reply:
Thanks for the review, nice catch. I tried to implement your suggestions.